# Service for stopping active movement
---
bool success
string message
